web-app/src/tests/App.test.js

```diff
@@ -0,0 +1,28 @@
+import puppeteer from 'puppeteer';
+import {APP_URL} from './app-url';
+
+/**
+ * PAGE LOADS SUCCESSFULLY
+ */
+describe('On page load', () => {
+    test('Home page loads correctly', async () => {
+        const browser = await puppeteer.launch({});
+        const page = await browser.newPage()
+
+        await page.emulate({
```

Comment:
Remove page.emulate - we dont need this.

Reply:
Noted sir.

Reply:
Noted sir.

Reply:
Done